tool/ood-gen/lib/rss.ml

```diff
@@ -31,6 +31,12 @@ let feeds () =
 
 let scrape () =
   let feeds = feeds () in
+  let id = Uri.of_string "https://ocaml.org/atom" in
+  let title : Syndic.Atom.title = Text "OCaml.org blog" in
+  let updated = Ptime.of_float_s (Unix.gettimeofday ()) |> Option.get in
+  let entries = feeds |> River.posts |> River.create_atom_entries in
+  let feed = Syndic.Atom.feed ~id ~title ~updated entries in
+  Syndic.Atom.write feed "data/rss/atom.xml";
```

Comment:
You can generate the file in `asset/feed.xml`, and the file will be available directly from https://ocaml.org/feed.xml
```suggestion
Syndic.Atom.write feed "asset/feed.xml";
```